File: 1. Programming basics with Java/IF & ELSE Statements/WorldSwimmingRecord.java
import java.util.Scanner;

public class WorldSwimmingRecord {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        double recordInSec = Double.parseDouble(scanner.nextLine()); // the world record in seconds
        double recordInMeters = Double.parseDouble(scanner.nextLine()); // the world record in meters
        double timeInSec = Double.parseDouble(scanner.nextLine()); // the time in seconds for one meter swam

        double timeNeeded = 0.0; double addedTime = 0.0; double totalTime = 0.0;

        timeNeeded = recordInMeters * timeInSec;
        addedTime = Math.floor((recordInMeters / 15)) * 12.5;
        totalTime = timeNeeded + addedTime;


        if (recordInSec < totalTime){
            System.out.printf("No, he failed! He was %.2f seconds slower.", totalTime - recordInSec);
        }else{
            System.out.printf("Yes, he succeeded! The new world record is %.2f seconds.", totalTime);
        }

    }
}

File: 9. JS Front-End/Conditional Statements/WorldSwimmingRecord.js
function swimmingRecord(input) {

    let recordInSeconds = Number(input[0]);
    let meters = Number(input[1]);
    let timeToSwimOneMeterInSeconds = Number(input[2]);

    let seconds = meters * timeToSwimOneMeterInSeconds;
    let addedSeconds = Math.floor((meters / 15)) * 12.5;

    let totalSeconds = seconds + addedSeconds;

    if (recordInSeconds < totalSeconds) {
        console.log(`No, he failed! He was ${Math.abs(recordInSeconds - totalSeconds).toFixed(2)} seconds slower.`)
    } else {
        console.log(`Yes, he succeeded! The new world record is ${totalSeconds.toFixed(2)} seconds.`)
    }
}

swimmingRecord(["55555.67",
"3017",
"5.03"])